Tutorial.ipynb

```diff
@@ -181,7 +185,7 @@
bR3qX4uPGCRJ2iVdJ/p1SW4G9gCeCrx4sCPJ/sCXgIOAc2ebzSc5BThliWKVJGnijNOt+2OAy4BDaiioJPsBnwdOqar75ziPt+4lScvIBK51X1XfBp4E7DOj/16aZ/THdhGXJEmTbGwSfZKDgZXAg0melmRd2/8E4EXA97uMT5KkSdT1/e7BM3qAAL9VVduTPBu4uLklT4D3VdXGzqKUJGlCWY9ekqSJM4HP6CVJ0u5nopckqcdM9JIk9ZiJXpKkHjPRS5LUYyZ6SZJ6bOSJPslpSapdEGfmvs8meUaStyU5c6j/Y23p2luSXNFWsJMkSTtpKWb0ZwLXte1M66vqLuB44Nqh/rdX1XOr6jDgH4GzRx+mJEn9M9JE387EXwS8AXjtUP+nk9wOHNyujPfrwJeSvBGgqh5qxwVYB0z+qj6SJHVg1MvJvRL4alX9IMmDSY6sqhur6qwkZwAHAFfQLHF7xvCBST4OnATcDrxzxHFKktRLo751fyZwefv75Uy/fX8EsAE4rG2nqarfBvYDvgf85mwnT3JKko/uzoAlSeqTka11n+SJwD3AT2huva9s27cAfwo8g+b5+z7AI8A/VtWJs5znOOBdVXXyPNdyrXtJ0jIyHmvdnw58qqqeXlXrq2p/4C7gYeBI4NaqOhS4DXjeIMmncdDgd+BU4P+MME5JknprlNPgM4H3zOj7XNv/U2BDkjXA6sGH71oBPplk7/b3DTR3ASRJ0k6yTK0kSRNnPG7dS5KkjpnoJUnqMRO9JEk9ZqKXJKnHTPSSJPWYiV6SpB4z0UuS1GNjkehn1qxP8vQkNyW5OcltSd7cdYySJE2isUj0/GLN+vuAY6rqcOAFwPlJ9usqOEmSJlXniX62mvVVtbWqtrRD1jIGcUqSNInGIYE+WrMeeDDJkQBJ9k9yC/Aj4D1Vde/MAy1TK0nS/MYh0c9as76qflRVhwEHAb+V5MkzD6yqq6vqTUsWqSRJE6bTSjBtzfoXA4c2hWmamvVJzq222k5V3ZvkVuBY4IruopUkafJ0PaOfq2b9sUnWASR5As0z/O93GKckSROp69quc9Ws/ytgWzvLD/C+qtq41MFJkjTprEcvSdLEsR69JEnCRC9JUq+Z6CVJ6jETvSRJPWailySpx0z0kiT12EgTfZInJ/lMkjuT3Jjk20leNWPMNUn2SPLBJMcM9X81yYa2TO2lSVaOMlZJkvpoZIk+SYDPA9dW1TOr6kia6nRPGxqzDthRVZuB5wPfHTrFa6rqucAhwD7AGaOKVZKkvhrljP7FwNaqunTQUVV3V9WHoJnJAxuBQ5JsBA4FbkhyUjv2ofawVcAaYPJX9pEkaYmNcjm55wA3zbWzqk5Mci5wJ/AAcHJVnTs8JsnfAkcDX8GCNpIk7bQl+zBekg+3z9xvGOo+AtgAHNa201TVy4GnAmtp7hDMPKf16CVJmsfI1rpP8hLg3VV1/FDfk2iew18EnE1Ta/4O4ADgfuCmqjprlnP9B+Doqjp7jmu51r0kaRlZ/Fr3o0z0Ab4DfKKqPtL2HUDz4bz1SfYFLquqU5NcX1VHDx27F/C4qrovySrg08A3q+qSOa71MJax7dqTaB7BqFu+D93zPRgPfX8fnl5V+yxm4MimwVVVSU4DPpDkXcBPgEeA89ohxwHXJdkfuHvG4b8EXJVkLc3jhWuAS5nb9xf7l41GI8l3fQ+65/vQPd+D8eD78HMjvd9dVffRfKVutn3DH647Y8a++2m+bidJkh4DV8aTJKnH+pLo/eR993wPxoPvQ/d8D8aD70NrZB/GkyRJ3evLjF6SJM1iohN9kn+X5PtJ7khyftfxLAcLveZJXp/kJ0lubn/e2EWcy02Sv07y4yS3dh3LcrHQa57khCT/b+i/hXcvdYzLTZL920Jpt7cF0d7adUzjYGJv3bfV7H4AvAy4B7gBOLOqbu80sB5bzGue5PXAUXMtbqTRSHIcsAn4m6o6pOt4loOFXvMkJwDnVNXJSx3bcpXkqcBTq+qmJI8DbgROW+55YZJn9EcDd1TVnVW1FbgceGXHMfWdr/mYqqprgX/pOo7lxNd8/FTVfVV1U/v7w8D3gF/tNqruTXKi/1XgR0Pb9+AbOmqLfc1/I8ktSa5oF0SSlqtj2hofX0nynK6DWU6SrAeeB/zvbiPp3iQneo2nq4H1VXUY8HXgkx3HI3XlJpplSp8LfAj4fMfxLBvtMuqfA942VPJ82ZrkRP9PwPBs8Wltn0Znwde8qh6sqi3t5mXAkUsUmzRWquqhqtrU/v5lYHVb2EsjlGQ1TZL/dFX9z67jGQeTnOhvAP5VkmckWUOz1O5VHcfUdwu+5u2HYQZOpXlGJi07SZ7SFvciydE0/799sNuo+q19vT8GfK+q3t91PONiYmu7VtVUkrOBvwVWAn9dVbd1HFavzfWaJ/kvwHer6irg95KcCkzRfFDp9Z0FvIwk+SxwAvCkJPcAf1xVH+s2qn6b7TUHVgNU1aXA6cBbkkwBPwNeW5P6NafJ8ULg3wMbk9zc9v1Be0dl2ZrYr9dJkqSFTfKte0mStAATvSRJPWailySpx0z0kiT1mIlekqQem9iv10na/ZL8CvCNdvMpwHbgJ+32T6vq1zoJTNIu8+t1kmaV5AJgU1W9r+tYJO06b91LWpQkm9r2hCR/n+QLSe5M8t+SnJXk+iQbkxzYjtsnyeeS3ND+vLDbf4G0PJnoJe2K5wJvBp5NsxLZs6rqaJr6Bv+5HfMXwAeq6vnAb7T7JC0xn9FL2hU3VNV9AEn+Afha278ROLH9/aXAv2mXewfYO8leg0IvkpaGiV7Srtgy9PuOoe0d/Pz/KyuAf1tVm5cyMEnTeete0qh8jZ/fxifJ4R3GIi1bJnpJo/J7wFFJbklyO80zfUlLzK/XSZLUY87oJUnqMRO9JEk9ZqKXJKnHTPSSJPWYiV6SpB4z0UuS1GMmekmSesxEL0lSj/1/d1wlwnWUcxUAAAAASUVORK5CYII=\n",
```

Comment:
Can you revert the changes to the data blobs? I think they're just noise.

Reply:
Yeah, my bad. Should be good now.